solution/ui/regulations/js/api.js

```diff
@@ -352,27 +375,33 @@ const getSupplementalContentLegacy = async (
     return result;
 };
 
-
-const getSupplementalContentByCategory = async (api_url, categories=[1,2]) =>{
+const getSupplementalContentByCategory = async (
+    api_url,
+    categories = [1, 2]
```

Comment:
Not your code but do we actually use the default for categories anywhere.  Having a hard coded one that is categories "1 and 2" can cause issues.  I dont think we ever do the call with no categories provided but I'm just curious if we ever do and why are we doing 1 and 2.

Reply:
not sure, but it's out of scope to debug a change like this.  This could be noted and collected in an omnibus tech debt story, like the formatting story you mentioned in slack.